src/main/features/dashboard/views/macro/claimStatus/claimFullAdmission.njk

```diff
@@ -0,0 +1,116 @@
+{% from "externalLink.njk" import externalLink %}
+{% from "form.njk" import linkButton %}
+
+{% macro claimFullAdmissionForDefendantDashboard(claim) %}
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
```

Comment:
I have seen this in more places, potential reuse possible to hold formatting payment method in one place. Filter might be an approach.